packages/shopify-cli-extensions/src/configs.ts

```diff
@@ -69,7 +69,7 @@ function isValidConfigs(
   requiredConfigs: RequiredConfigs,
   paths: string[] = [],
 ): configs is ConfigFile {
-  Object.keys(requiredConfigs).forEach((key) => {
+  Object.keys(requiredConfigs || {}).forEach((key) => {
```

Comment:
Why do we need this? Should it be set as a fallback then?
```ts
function isValidConfigs(
  configs: any,
  requiredConfigs: RequiredConfigs = {},
  paths: string[] = [],
)
```

Reply:
@kumar303 do you think we still need this? I know we ran into an issue here without it, but that might have been due to our `localization: null` erroneously being spat out into the config file.

Reply:
We were getting `TypeError: Cannot convert undefined or null to object` but I'm not sure why. It kept happening even after we removed `localization: null` manually from the config file. It could be that something is setting `requiredConfigs` to `null` 🤷 

It's annoying that `Object.keys()` throws like this so it's probably a good practice to always fallback with `|| {}`.